src/database/migration/file/index.ts

```diff
@@ -17,6 +9,18 @@ import MigrationVersion6 from './migration_version_6';
 import MigrationVersion7 from './migration_version_7';
 import MigrationVersion8 from './migration_version_8_change_enum_to_text';
 import MigrationVersion9 from './migration_version_9_users_add_anon_user_info_emoji_code';
+import MigrationVersion10 from './migration_version_10_users_add_anon_user_info_color_name';
+import MigrationVersion11 from './migration_version_11_users_add_anon_user_info_color_code';
+import MigrationVersion12 from './migration_version_12_channel_lists_add_anon_user_info_emoji_name';
+import MigrationVersion13 from './migration_version_13_channel_lists_add_anon_user_info_emoji_code';
+import MigrationVersion14 from './migration_version_14_channel_lists_add_anon_user_info_color_name';
+import MigrationVersion15 from './migration_version_15_channel_lists_add_anon_user_info_color_code';
+import MigrationVersion16 from './migration_version_16_users_add_anon_user_info_emoji_name';
+import MigrationVersion17 from './migration_version_17_users_add_is_anonymous';
+import MigrationVersion18 from './migration_version_18_channel_list_add_column_type_new';
+import MigrationVersion19 from './migration_version_19_channel_list_copy_column_type_to_column_type_new';
+import MigrationVersion20 from './migration_version_20_channel_list_delete_channel_type';
+import MigrationVersion21 from './migration_version_21_channel_list_rename_column_type_new';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new migration files are imported correctly. However, it's important to ensure that these migrations are idempotent and can be run multiple times without causing issues. Also, the order of migrations matters. Make sure they are in the correct order and each migration depends only on the ones before it.
```diff
+ import MigrationVersion10 from './migration_version_10_users_add_anon_user_info_color_name';
+ import MigrationVersion11 from './migration_version_11_users_add_anon_user_info_color_code';
+ import MigrationVersion12 from './migration_version_12_channel_lists_add_anon_user_info_emoji_name';
+ import MigrationVersion13 from './migration_version_13_channel_lists_add_anon_user_info_emoji_code';
+ import MigrationVersion14 from './migration_version_14_channel_lists_add_anon_user_info_color_name';
+ import MigrationVersion15 from './migration_version_15_channel_lists_add_anon_user_info_color_code';
+ import MigrationVersion16 from './migration_version_16_users_add_anon_user_info_emoji_name';
+ import MigrationVersion17 from './migration_version_17_users_add_is_anonymous';
+ import MigrationVersion18 from './migration_version_18_channel_list_add_column_type_new';
+ import MigrationVersion19 from './migration_version_19_channel_list_copy_column_type_to_column_type_new';
+ import MigrationVersion20 from './migration_version_20_channel_list_delete_channel_type';
+ import MigrationVersion21 from './migration_version_21_channel_list_rename_column_type_new';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->